AndelaOSP/art-android
# Add dialog for adding serial and fetching the asset

**What does this PR do?**
Enables fetching asset data when serial number is entered.

**Description of Task to be completed?**
- Create serial entry dialog
- Fetch data from api using serial

**How should this be manually tested?**
Click on the check serial button and type a serial number when the dialog pops up . You will be redirected to the checkin activity

**Any background context you want to provide?**
Since the endpoint is not yet created you will have to use json web server and ngrok to make a fake web server then use the generated url as your base url.

**Screenshots**
![screenshot_1522938091](https://user-images.githubusercontent.com/5074111/38371890-3bbb2d72-38f6-11e8-9363-90f84f10147b.png)
![screenshot_1522938098](https://user-images.githubusercontent.com/5074111/38371892-3c15e118-38f6-11e8-997b-88b5de8719fa.png)
![screenshot_1522938110](https://user-images.githubusercontent.com/5074111/38371893-3c5ef7c2-38f6-11e8-83cf-82c247d17eae.png)
